README.md

```diff
@@ -50,3 +56,5 @@ This will make the release available for use. For subsequent releases:
         git pull
 
 2.  Follow the above steps from Step 3 on.
+
+**Note**: Tagging the version will be handled by the maintainers and should be ignored by producers.
```

Comment:
"Ignored" isn't the right word here. Tags are important for producers because producers should be using the tagged version of their schema.